src/Goldilocks/Goldilocks.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="$(MicrosoftAspNetCoreAppPackageVersion)" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
+    <PackageReference Include="System.Text.Json" Version="$(MicrosoftAspNetCoreAppPackageVersion)" />
```

Comment:
It's part of net8.0, shouldn't be necessary

Reply:
Fixed the unnecessary dependency.